roles/bridge/tasks/main.yml

```diff
@@ -31,6 +32,7 @@
     - Restart service
 
 - name: Fetch secrets from HCP Vault and save to files
+  when: fetch_validator_keys | default(false) | bool
```

Comment:
lets rename to fetch_bridge_keys or we can omit this variable, a bridge instance can run with two keys, so this variable isnt really needed

Reply:
Ok